### PR TITLE
First draft of roadmap section in README

### PR DESCRIPTION
**Pull request by kenperkins**
This is a first draft of a rough roadmap. I decided to add to the README so it's always obvious without having to ask users to browse around the repo.

Thoughts?

### CODE REVIEW

**Comment by kenperkins** [comment]
This will close #206

**Comment by jrperritt** [comment]
lgtm

**Comment by kenperkins** [comment]
This is ready for review.

**Comment by jrperritt** [comment]
+2

File: README.md [hunk 1]
```diff
@@ -65,3 +65,31 @@ and source it:
...
+7. RackConnect
+8. Cloud Big Data \*
+9. Cloud Monitoring \*
+10. Cloud Orchestration
+11. Cloud Queues \*
+12. Cloud Backup \*
+13. Autoscale \*
+14. Cloud Metrics \*
+
+\* Services not supported in [Gophercloud](github.com/rackspace/gophercloud) at present.
```
Comment (rgbkrk) [inline README.md]: Should be https://github.com/rackspace/gophercloud otherwise this will be interpreted as a relative link.

**Pull request merged**